### PR TITLE
Contributors and Contact: point to same section in se-edu #83

### LINKED ISSUE #83: Contributors/Contact: point to same section in se-edu
The sections in concern can point to the same sections in se-edu, rather than duplicate here.
Make sure all the names here are listed in se-edu page as well.

### PR DESCRIPTION
**Pull request by limmlingg**
Fixes #83

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -459,13 +459,9 @@ Some suggested enhancements:
...
-* [Damith C. Rajapakse](http://www.comp.nus.edu.sg/~damithch): Project Advisor
+The full list of contributors for se-edu can be found [here](https://se-edu.github.io/docs/Team.html).
 
 -----------------------------------------------------------------------------------------------------
 # Contact Us
 
-* **Bug reports, Suggestions**: Post in our [issue tracker](https://github.com/se-edu/addressbook-level1/issues)
-  if you noticed bugs or have suggestions on how to improve.
-* **Contributing**: We welcome pull requests. Follow the process described [here](https://github.com/oss-generic/process)
+If you would like to contribute to se-edu or contact us, refer to [our website](http://se-edu.github.io/).
```
Comment (yamgent) [inline README.md]: Should it be pointing specifically to https://se-edu.github.io/#contributing and https://se-edu.github.io/#contact?

**Comment by CanIHasReview** [comment]
# v1

@limmlingg submitted v1 for review.

* [v1 1/1] [Contributors and Contact: point to same section in se\-edu](https://github.com/se-edu/addressbook-level1/pull/86/commits/b306e4e321b65b5baf435f5fd789b982bac35f5d)

([:books: Archive](https://github.com/se-edu/addressbook-level1/compare/a894d3fd42492288072420720536031398c4bc75...b306e4e321b65b5baf435f5fd789b982bac35f5d))
<details>
<summary>Checkout this PR version locally</summary>

```sh
git fetch https://github.com/se-edu/addressbook-level1.git refs/pr/86/1/head:BRANCHNAME
```
where `BRANCHNAME` is the name of the local branch you wish to fetch this PR to.

</details>

File: README.md [hunk 2]
```diff
@@ -459,13 +459,11 @@ Some suggested enhancements:
...
-* [Damith C. Rajapakse](http://www.comp.nus.edu.sg/~damithch): Project Advisor
+The full list of contributors for se-edu can be found [here](https://se-edu.github.io/docs/Team.html).
 
 -----------------------------------------------------------------------------------------------------
 # Contact Us
 
 * **Bug reports, Suggestions**: Post in our [issue tracker](https://github.com/se-edu/addressbook-level1/issues)
   if you noticed bugs or have suggestions on how to improve.
-* **Contributing**: We welcome pull requests. Follow the process described [here](https://github.com/oss-generic/process)
+* If you would like to contact us, refer to [our website](https://se-edu.github.io/#contact).
```
Comment (damithc) [inline README.md]: No more mention of contributing?

**Comment by CanIHasReview** [comment]
# v2

@limmlingg submitted v2 for review.

* [v2 1/1] [Contributors and Contact: point to same section in se\-edu](https://github.com/se-edu/addressbook-level1/pull/86/commits/a9de2c3874ccfd633d444be63024662eaec7a33b)

([:books: Archive](https://github.com/se-edu/addressbook-level1/compare/a894d3fd42492288072420720536031398c4bc75...a9de2c3874ccfd633d444be63024662eaec7a33b)) ([:chart_with_upwards_trend: Interdiff between v1 and v2](https://github.com/se-edu/addressbook-level1/compare/84eebac621a3ae2229c569a9e3e8dcb72f35170f~1...84eebac621a3ae2229c569a9e3e8dcb72f35170f))
<details>
<summary>Checkout this PR version locally</summary>

```sh
git fetch https://github.com/se-edu/addressbook-level1.git refs/pr/86/2/head:BRANCHNAME
```
where `BRANCHNAME` is the name of the local branch you wish to fetch this PR to.

</details>

**Pull request merged**